From 7f8ca0ae0b89f4f4442bf4f7f05677f255121dd6 Mon Sep 17 00:00:00 2001
From: Brad Bishop <bradleyb@fuzziesquirrel.com>
Date: Wed, 14 Mar 2018 16:03:38 -0400
Subject: [PATCH] syscfg: Add ppc64le cross build support.

* src/syscfg/lock-obj-pub.ppc64le-unknown-linux-gnu.h: New.
* src/Makefile.am (lock_obj_pub): Add.
--

Signed-off-by: Brad Bishop <bradleyb@fuzziesquirrel.com>
---
 src/Makefile.am                                    |  1 +
 .../lock-obj-pub.ppc64le-unknown-linux-gnu.h       | 25 ++++++++++++++++++++++
 2 files changed, 26 insertions(+)
 create mode 100644 src/syscfg/lock-obj-pub.ppc64le-unknown-linux-gnu.h

diff --git a/src/Makefile.am b/src/Makefile.am
index 268c2ab..e3ed6c1 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -64,6 +64,7 @@ lock_obj_pub = \
         syscfg/lock-obj-pub.powerpc-unknown-linux-gnu.h     \
         syscfg/lock-obj-pub.powerpc64-unknown-linux-gnu.h   \
 	syscfg/lock-obj-pub.powerpc64le-unknown-linux-gnu.h \
+	syscfg/lock-obj-pub.ppc64le-unknown-linux-gnu.h     \
 	syscfg/lock-obj-pub.powerpc-unknown-linux-gnuspe.h  \
 	syscfg/lock-obj-pub.riscv64-unknown-linux-gnu.h     \
         syscfg/lock-obj-pub.s390x-ibm-linux-gnu.h           \
diff --git a/src/syscfg/lock-obj-pub.ppc64le-unknown-linux-gnu.h b/src/syscfg/lock-obj-pub.ppc64le-unknown-linux-gnu.h
new file mode 100644
index 0000000..b9407d4
--- /dev/null
+++ b/src/syscfg/lock-obj-pub.ppc64le-unknown-linux-gnu.h
@@ -0,0 +1,25 @@
+## lock-obj-pub.ppc64le-unknown-linux-gnu.h
+## File created by gen-posix-lock-obj - DO NOT EDIT
+## To be included by mkheader into gpg-error.h
+
+typedef struct
+{
+  long _vers;
+  union {
+    volatile char _priv[40];
+    long _x_align;
+    long *_xp_align;
+  } u;
+} gpgrt_lock_t;
+
+#define GPGRT_LOCK_INITIALIZER {1,{{0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0}}}
+##
+## Local Variables:
+## mode: c
+## buffer-read-only: t
+## End:
+##
-- 
2.5.0

